### PR TITLE
BusinessHours default times

### PR DESCRIPTION
**Pull request by rafalpospiech**
obraz

obraz

### CODE REVIEW

File: install/install_schema/scheme.sql [hunk 1]
```diff
@@ -997,6 +997,9 @@ CREATE TABLE `s_yf_business_hours` (
   `working_hours_from` varchar(8) NOT NULL DEFAULT '00:00:00',
   `working_hours_to` varchar(8) NOT NULL DEFAULT '00:00:00',
   `holidays` tinyint(1) NOT NULL DEFAULT 0,
+	`reaction_time` varchar(20) NOT NULL DEFAULT '00:m',
```
Comment (mariuszkrzaczkowski) [inline install/install_schema/scheme.sql]:
```suggestion
	`reaction_time` varchar(20) NOT NULL DEFAULT '0:m',
```

File: public_html/layouts/resources/Fields.js [hunk 2]
```diff
@@ -1779,5 +1782,132 @@ window.App.Fields = {
...
+			this.container = container;
+			this.value = container.val();
+			if (this.value) {
+				const split = this.value.split(':');
+				this.time = Number(split[0]);
+				this.period = split[1];
+			} else {
+				this.time = 0;
+				this.period = 'm';
+				this.value = '00:m';
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/resources/Fields.js]:
```suggestion
				this.value = '0:m';
```

File: install/install_schema/scheme.sql [hunk 3]
```diff
@@ -997,6 +997,9 @@ CREATE TABLE `s_yf_business_hours` (
...
+  `idle_time` varchar(20) NOT NULL DEFAULT '00:m',
```
Comment (mariuszkrzaczkowski) [inline install/install_schema/scheme.sql]:
```suggestion
  `idle_time` varchar(20) NOT NULL DEFAULT '0:m',
```

File: install/install_schema/scheme.sql [hunk 4]
```diff
@@ -997,6 +997,9 @@ CREATE TABLE `s_yf_business_hours` (
...
+  `resolve_time` varchar(20) NOT NULL DEFAULT '00:m',
```
Comment (mariuszkrzaczkowski) [inline install/install_schema/scheme.sql]:
```suggestion
  `resolve_time` varchar(20) NOT NULL DEFAULT '0:m',
```

File: layouts/basic/modules/Settings/BusinessHours/EditViewBlocks.tpl [hunk 5]
```diff
@@ -17,17 +17,14 @@
 				</div>
 			</div>
 			<div class="card">
-				<div class="card-header">{if !empty($RECORD_MODEL->getId())}{\App\Language::translate('LBL_EDIT_BUSINESS_HOURS',$QUALIFIED_MODULE)} - {$RECORD_MODEL->getName()}{else}{\App\Language::translate('LBL_ADD_BUSINESS_HOURS',$QUALIFIED_MODULE)}{/if}</div>
+				<div class="card-header">{if !empty($RECORD_MODEL->getId())}<span class="fas fa-edit mr-2"></span>{\App\Language::translate('LBL_EDIT_BUSINESS_HOURS',$QUALIFIED_MODULE)} - {$RECORD_MODEL->getName()}{else}<span class="fas fa-plus mr-2"></span>{\App\Language::translate('LBL_ADD_BUSINESS_HOURS',$QUALIFIED_MODULE)}{/if}</div>
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/BusinessHours/EditViewBlocks.tpl]:
```suggestion
				<div class="card-header">
				{if !empty($RECORD_MODEL->getId())}
				<span class="fas fa-edit mr-2"></span> {\App\Language::translate('LBL_EDIT_BUSINESS_HOURS',$QUALIFIED_MODULE)} - {$RECORD_MODEL->getName()}
				{else}
				<span class="fas fa-plus mr-2"></span>{\App\Language::translate('LBL_ADD_BUSINESS_HOURS',$QUALIFIED_MODULE)}{/if}
				</div>
```

File: public_html/layouts/resources/Fields.js [hunk 6]
```diff
@@ -1779,5 +1782,132 @@ window.App.Fields = {
...
+
+		/**
+		 * On change event
+		 *
+		 * @param {Event} event
+		 */
+		onChange(event) {
+			this.time = this.input.val();
+			this.period = this.select.val();
+			this.value = this.input.val().padStart(2, '0') + ':' + this.select.val();
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/resources/Fields.js]: ?

File: public_html/layouts/basic/styles/components/_Global.scss [hunk 7]
```diff
@@ -558,3 +558,21 @@ input[type='checkbox'][readonly],
 input[type='checkbox'][disabled] {
 	background-color: material-color('grey', '200');
 }
+
+/* -------- TIME PERIOD FIELD -------- */
+.js-time-period-input::-webkit-outer-spin-button,
```
Comment (rskrzypczak) [inline public_html/layouts/basic/styles/components/_Global.scss]: in .scss no js- class

**Pull request merged**